source/contributing.rst

```diff
@@ -86,18 +86,21 @@ and rebuild entirely: ::
    $ rm -r docs
    $ python source/quickguides/generate_quick_guides.py # Run if quick-guide files were updated
    $ make html # Make sure you have run `conda activate sphinx` first
- 
 
-Once you are satisfied with your modifications, commit and push them back to 
-the repository: ::
 
-   $ cd <myDir>/e3sm_diags
-   $ git add .
-   $ git commit
-   $ git push <your-fork-remote-name> <branch-name> # If not using a fork, use `origin`
+Once you are satisfied with your modifications, commit and push them back to
+the repository:
+
+::
+
+  $ cd <myDir>/e3sm_diags
+  $ git add .
+  # --no-verify is required to bypass pre-commit checks until docs are moved over to the master branch
```

Comment:
This is the main change for this file. The other changes were from PyCharm's formatting.